force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +87,21 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> allWordsList = sentence.split(' ');
         // Create a set to store the unique words
-
+        Set<String> uniqueWordsSet = new Set<String>();
         // Add each word to the set
-
+        for (string word : allWordsList) {
+            uniqueWordsSet.add(word);    
+            }
         // Join the unique words back into a sentence
-
+        String outputSentence = '';
+        Integer i = 1;
+        for (string uniqueWord : uniqueWordsSet) {
+            outputSentence = outputSentence + uniqueWord + ' ';
+        }
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return outputSentence.trim(); // Replace null with the variable you used to store the result
```

Comment:
If you use the `join()` method, you won't need to trim the final result. Otherwise, there's nothing wrong with your code -- it simply can be shortened using available methods.